package.json

```diff
@@ -53,5 +54,7 @@
   "volta": {
     "node": "16.17.0",
     "npm": "8.18.0"
-  }
+  },
+  "readme": "ERROR: No README data found!",
+  "_id": "the-charity-frontend@0.0.0"
```

Comment:
What are these for?

Reply:
what the heck
I didn't see them, must have been chromatic which added these
I'll remove them